src/Workspaces/Core/Portable/CodeFixesAndRefactorings/IFixAllContext.cs

```diff
@@ -14,13 +14,7 @@ namespace Microsoft.CodeAnalysis.CodeFixesAndRefactorings;
 internal interface IFixAllContext
 {
     IFixAllState State { get; }
-    IFixAllProvider FixAllProvider { get; }
-    Solution Solution { get; }
-    Project Project { get; }
-    Document? Document { get; }
     object Provider { get; }
-    FixAllScope Scope { get; }
-    string? CodeActionEquivalenceKey { get; }
```

Comment:
all tehse removed values are present in IFixAllState.  It represents the snapshot of the world that hte ephemeral context (which contains progress/cancellation) wraps.